data/scripts.js

```diff
@@ -1381,7 +1381,7 @@ exports.BattleScripts = {
 			item = template.requiredItem;
 		} else if (template.species === 'Rotom-Fan') {
 			// this is just to amuse myself
-			// do we really have to keep this
+			// for the sake of the world's balance, this must always stay
```

Comment:
The new comment doesn't make sense given the context.
Just remove the <code>// do we really have to keep this</code>, or add the world's balance comment below.